packages/caip/src/caip2/caip2.ts

```diff
@@ -19,6 +20,7 @@ export enum ChainReference {
   // caip2 uses max length of 32 chars of the genesis block
   BitcoinMainnet = '000000000019d6689c085ae165831e93',
   BitcoinTestnet = '000000000933ea01ad0ee984209779ba',
+  CosmosMainnet = 'cosmoshub-4',
   CosmosHubMainnet = 'cosmoshub-4',
```

Comment:
Seems like there was another caip2 merged through recently that now is kind of duplicated with what was in place here. that we should clean up